app/src/androidTest/java/com/example/avjindersinghsekhon/minimaltodo/testApp/tests/StartTheApp.java

```diff
@@ -0,0 +1,94 @@
+package com.example.avjindersinghsekhon.minimaltodo.testApp.tests;
+
+import android.support.test.espresso.Espresso;
+
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+import static junit.framework.Assert.assertTrue;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class StartTheApp extends EspressoTestBase {
```

Comment:
Add a name to reflect what these tests are validating

Reply:
Done.